interfaces/builtin/broadcom_asic_control.go

```diff
@@ -49,9 +40,11 @@ const broadcomAsicControlConnectedPlugAppArmor = `
 `
 
 const broadcomAsicControlConnectedPlugUDev = `
-KERNEL=="linux-user-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-kernel-bde", TAG+="###SLOT_SECURITY_TAGS###"
-KERNEL=="linux-bcm-knet", TAG+="###SLOT_SECURITY_TAGS###"
+# This file contains udev rules for accessing broadcom asic kernel module.
+#
+# Do not edit this file, it will be overwritten on updates
+
+SUBSYSTEM=="net", KERNEL=="bdev", TAG+="%[1]s"
```

Comment:
Here is the udev rule from upstream,  
https://git.openswitch.net/cgit/openswitch/ops-build/commit/?id=c0e324d0e76e9464f115434cce74301b5794ff61

I'm afraid it doesn't work that way if we use kernel *module* name rather than the 'bdev' (bcm device) in this case.